modules/home-environment.nix

```diff
@@ -98,14 +98,12 @@ in
   options = {
     home.username = mkOption {
       type = types.str;
-      default = builtins.getEnv "USER";
```

Comment:
Could keep `defaultText = "getEnv \"USER\"";` or `defaultText = "$USER";` maybe? Same for HOME below.